### PR TITLE
NETOBSERV-229 create pre-merge image on PR via action

### PR DESCRIPTION
**Pull request by jotak**
Uses the 'ok-to-test' label to trigger the build/push

### CODE REVIEW

File: .github/workflows/push_image_pr.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      - uses: actions/github-script@v5
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'New image: ${{ steps.push-to-quay.outputs.registry-paths }}. It will expire after two weeks.'
+            })
```
Comment (ronensc) [inline .github/workflows/push_image_pr.yml:57]: I'm a bit confused. If this workflow is triggered by a PR, then what is the issue in context where the comment is created?

Comment (jotak) [reply .github/workflows/push_image_pr.yml:57]: in github API, a PR is apparently some special kind of issue. But the comment is actually created on the PR

**Review by eranra — COMMENTED** [review]
@jotak looks good to me

**Review by eranra — APPROVED** [review]

**Comment by jotak** [comment]
thanks

**Pull request merged**